cardano-cli/test/cardano-cli-test-lib/Test/Cardano/CLI/Util.hs

```diff
@@ -82,7 +83,20 @@ execDetailCardanoCLI
   -- ^ Arguments to the CLI command
   -> m (IO.ExitCode, String, String)
   -- ^ Captured stdout
-execDetailCardanoCLI = GHC.withFrozenCallStack $ execDetailFlex H.defaultExecConfig "cardano-cli" "CARDANO_CLI"
+execDetailCardanoCLI params = GHC.withFrozenCallStack $ execDetailCfgCardanoCLI H.defaultExecConfig params
+
+-- | Execute cardano-cli via the command line, expecting it to fail, and accepting custom config.
+--
+-- Waits for the process to finish and returns the exit code, stdout and stderr.
+execDetailCfgCardanoCLI
```

Comment:
`execDetailConfigCardanoCLI` <- don't shorten the name

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/895/commits/09affb66b15d9691f6d0541d5b4e6a892edf14fe